tests/e2e/web/src/App.test.tsx

```diff
@@ -23,4 +27,11 @@ describe('ESM browser validation', () => {
     // check the sdk loaded the data
     expect(result).toHaveTextContent('true');
   });
+
+  it('should retrieve ids from registry', async () => {
+    const affiliatedAnts = await registry.accessControlList({ address });
+
+    expect(Array.isArray(affiliatedAnts.Owned)).toEqual(true);
+    expect(Array.isArray(affiliatedAnts.Controlled)).toEqual(true);
```

Comment:
the goal of this test should be to validate that the registry component can be used in `App.tsx` - refer to the other test for how to align

Reply:
https://github.com/ar-io/ar-io-sdk/pull/177/commits/38ca913fc9f935663b750e4f68e97879d3b70e5f

Reply:
...this test is not checking the behavior of successful call to the registry on App.tsx - please look at the test above

Reply:
Gotchya, updated to use the render logic.